src/themes/BootstrapTheme/Menu/AdminMenu.php

```diff
@@ -21,7 +21,7 @@ class AdminMenu implements ContainerAwareInterface
     use ContainerAwareTrait;
     use TranslatorTrait;
 
-    public function adminMenu(FactoryInterface $factory, array $options)
+    public function menu(FactoryInterface $factory, array $options)
```

Comment:
you must change `src/themes/BootstrapTheme/Resources/views/Include/admin_menu.html.twig`